nbgrader_to_canvas/models.py

```diff
@@ -0,0 +1,39 @@
+from nbgrader_to_canvas import db
+
+
+# DB Model
```

Comment:
using raw db objects directly in business logic may lead to a lot of refactoring if the db schema changes or the db backend changes